### PR TITLE
Placeholder ember-cli meeting notes 19 oct 2017

### PR DESCRIPTION
**Pull request by hjdivad**

### CODE REVIEW

File: ember-cli/2017-10/october-19.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# ember-cli Meeting Notes - 2017-10-19
+
+## Attendees
+
+
+## Agenda
```
Comment (hjdivad) [inline ember-cli/2017-10/october-19.md]: agenda here

Comment (rwjblue) [reply ember-cli/2017-10/october-19.md]: ember-qunit update

Comment (rwjblue) [reply ember-cli/2017-10/october-19.md]: strategies RFCs

Comment (rwjblue) [reply ember-cli/2017-10/october-19.md]: ember-cli-update

Comment (kratiahuja) [reply ember-cli/2017-10/october-19.md]: Where does Ember CLI use this library?

Comment (rwjblue) [reply ember-cli/2017-10/october-19.md]: console-ui

Comment (kratiahuja) [reply ember-cli/2017-10/october-19.md]: console-ui has picked a non MIT license : https://github.com/aadsm/jschardet/blob/master/LICENSE in Ember 2.16. The bump was from greenkeeper. should we bump down and release 2.16.3?

**Comment by hjdivad** [comment]
cc @kategengler @kellyselden @kratiahuja @rwjblue @trabus @turbo87

**Pull request merged**